src/resources/scss/components/_card.scss

```diff
@@ -27,4 +27,45 @@
 
 .card > .card-body + .list-group > .list-group-item:first-of-type {
     border-top: 1px solid $list-group-border-color;
-}
\ No newline at end of file
+}
```

Comment:
This changes has nothing do to with your PR, you should make PR only about a single topic. Please reset this file to master.